persistence-cassandra-3.11/pom.xml

```diff
@@ -52,6 +52,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
If this is temporary, no problem, but if it was to stay could definitely use a comment explaining the need for inclusion.

Reply:
Removed and hard-coded the version.